src/BLSSignatureChecker.sol

```diff
@@ -183,22 +193,30 @@ contract BLSSignatureChecker is IBLSSignatureChecker {
          */
         {
             bool _staleStakesForbidden = staleStakesForbidden;
-            uint256 withdrawalDelayBlocks = _staleStakesForbidden ? delegation.minWithdrawalDelayBlocks() : 0;
+            /// TODO: FIX
```

Comment:
will take a look

Reply:
Addressed here [a1295fa](https://github.com/Layr-Labs/eigenlayer-middleware/pull/333/commits/a1295fa14f2434ba73df4307d20140fd51cdfd02)

This does seem like a breaking change for core to make though since the selector for the minWithdrawal delay changed.  Any AVS that uses staleStakesForBidden will have their checkSignatures function bricked when the core protocol performs it's upgrade